<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameEndWidget</class>
 <widget class="QWidget" name="GameEndWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>570</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GO HOME</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>490</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>NEXT GAME</string>
   </property>
  </widget>
  <widget class="QLabel" name="heroImg">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>221</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rankText">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>571</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>YOUR FINAL RANK</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rankNumber">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>271</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
